test/end-to-end/pages/screeners/mentalcapacity.js

```diff
@@ -1,19 +1,24 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/mentalcapacity');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
+const mentalCapacityEn = require('app/resources/en/translation/screeners/mentalcapacity');
+const mentalCapacityCy = require('app/resources/cy/translation/screeners/mentalcapacity');
+const pageUnderTest = require('app/steps/ui/screeners/mentalcapacity');
```

Comment:
This is undoing performance work - including these files takes time - do we need to?